From 8eb27f87b78ec4a16a6aaf814f868c774f70546e Mon Sep 17 00:00:00 2001
From: Ken Ma <make@marvell.com>
Date: Mon, 28 May 2018 16:21:11 +0800
Subject: [PATCH 0681/1200] arm64: mvebu: a8k: config: move CONFIG_E1000 to
 defconfig files

This patch removes the hard coded dependency of CONFGI_PCIE_DW_MVEBU to
CONFIG_E1000, and moves CONFIG_E1000 to a8k defconfig files.

Change-Id: I586aab56c340c90ac2dcfa1ad07af81e1597870a
Signed-off-by: Ken Ma <make@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/55763
Reviewed-by: Igal Liberman <igall@marvell.com>
Tested-by: Igal Liberman <igall@marvell.com>
---
 configs/mvebu_db_armada8k_defconfig   | 1 +
 configs/mvebu_mcbin-88f8040_defconfig | 1 +
 include/configs/mvebu_armada-8k.h     | 7 -------
 3 files changed, 2 insertions(+), 7 deletions(-)

diff --git a/configs/mvebu_db_armada8k_defconfig b/configs/mvebu_db_armada8k_defconfig
index 8a54a3c7c6..d0bbc94b21 100644
--- a/configs/mvebu_db_armada8k_defconfig
+++ b/configs/mvebu_db_armada8k_defconfig
@@ -54,6 +54,7 @@ CONFIG_SPI_FLASH_SPANSION=y
 CONFIG_SPI_FLASH_STMICRO=y
 CONFIG_PHY_MARVELL=y
 CONFIG_PHY_GIGE=y
+CONFIG_E1000=y
 CONFIG_MVPP2=y
 CONFIG_PCI=y
 CONFIG_DM_PCI=y
diff --git a/configs/mvebu_mcbin-88f8040_defconfig b/configs/mvebu_mcbin-88f8040_defconfig
index 5d137cea63..b267422532 100644
--- a/configs/mvebu_mcbin-88f8040_defconfig
+++ b/configs/mvebu_mcbin-88f8040_defconfig
@@ -51,6 +51,7 @@ CONFIG_SPI_FLASH_STMICRO=y
 CONFIG_SPI_FLASH_WINBOND=y
 CONFIG_PHY_MARVELL=y
 CONFIG_PHY_GIGE=y
+CONFIG_E1000=y
 CONFIG_MVPP2=y
 CONFIG_PCI=y
 CONFIG_DM_PCI=y
diff --git a/include/configs/mvebu_armada-8k.h b/include/configs/mvebu_armada-8k.h
index d980fff28a..56f96cc9dd 100644
--- a/include/configs/mvebu_armada-8k.h
+++ b/include/configs/mvebu_armada-8k.h
@@ -26,11 +26,4 @@
 
 #define CONFIG_USB_MAX_CONTROLLER_COUNT (3 + 3)
 
-/*
- * PCI configuration
- */
-#ifdef CONFIG_PCIE_DW_MVEBU
-#define CONFIG_E1000
-#endif
-
 #endif /* _CONFIG_MVEBU_ARMADA_8K_H */
-- 
2.22.0

